packages/@react-aria/i18n/docs/useCollator.mdx

```diff
@@ -0,0 +1,43 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-aria/i18n';
+import {ClassAPI, HeaderInfo, TypeContext, FunctionAPI, TypeLink} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```tsx import
+import {useCollator} from '@react-aria/i18n';
+```
+
+---
+category: Internationalization
+---
+
+# useCollator
+
+<p>{docs.exports.useCollator.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['useCollator']} />
+
+## Introduction
+
+`useCollator` wraps a builtin browser [Intl.Collator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator)
+object to provide a React Hook that integrates with the i18n system in React Aria. It handles string comparison according to the current locale,
+updating when the locale changes, and caching of collators for performance. See the 
+[Intl.Collator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator) docs for 
+information.
+
```

Comment:
Though it may be trivial, what do you think about adding an example here? Maybe something as simple as 
```
let collator = useCollator({usage: 'search', sensitivity: 'base'});
alert(`z comes before a? ${collator.compare('z', 'a') > 0}`)
```

Reply:
added an example